components/projectsView/ProjectsList.js

```diff
@@ -10,8 +10,8 @@ import IndexIssueListItem from "../shared/IndexIssueListItem";
 import ProjectFilters from "./ProjectFilters";
 import ProjectScoreChart from "../shared/ProjectScoreChart";
 
-function includedInArray(issue, field, arr) {
-  return issue[field] ? arr.includes(issue[field]) : false;
+function includedInStringOrArray(issue, field, value) {
+  return issue[field] ? value.includes(issue[field]) : false;
```

Comment:
The way we're using this, we're asking if the substring issue[field] is inside the /string/ `value`. It's not wrong, because .includes() is a method on both strings and arrays, but this is a little confusing. I've renamed the function to be more explicit.